akka-actor/src/main/scala/akka/io/TcpConnection.scala

```diff
@@ -189,7 +189,7 @@ private[io] abstract class TcpConnection(val tcp: TcpExt, val channel: SocketCha
 
   /** used in subclasses to start the common machinery above once a channel is connected */
   def completeConnect(registration: ChannelRegistration, commander: ActorRef,
-                      options: immutable.Traversable[SocketOption]): Unit = {
+    options: immutable.Traversable[SocketOption]): Unit = {
```

Comment:
I thought that each parameter starts on a new line if there's one single break in the parameter list. Why doesn't that apply here? What's the logic behind this?